torch_np/_ndarray.py

```diff
@@ -112,7 +112,22 @@ def base(self):
     @property
     def flags(self):
         # Note contiguous in torch is assumed C-style
-        return Flags({"C_CONTIGUOUS": self._tensor.is_contiguous()})
+
+        # check if F contiguous
+        from itertools import accumulate
+
+        f_strides = tuple(accumulate(list(self._tensor.shape), func=lambda x, y: x * y))
```

Comment:
this is of course just `torch.cumprod(self._tensor.shape, 0)`; my instinct from numpy usage is to avoid array operations for small tuples etc.